src/test/java/pivotal/hooks/CommonHooks.java

```diff
@@ -0,0 +1,14 @@
+package pivotal.hooks;
+
+import core.selenium.WebDriverManager;
+import io.cucumber.java.After;
+
+public class CommonHooks {
```

Comment:
Sometimes it is better to avoid this kind of names because all the methods that don't belog to any class will be created here and this class will be too long